spaceship/lib/spaceship/portal/app.rb

```diff
@@ -15,6 +27,10 @@ class App < PortalBase
       # @return (String) the supported platform of this app
       # @example
       #   "ios"
+      #   "mac"
```

Comment:
To also go with this: http://appleinsider.com/articles/16/03/30/os-x-10114-hidden-framework-hints-apple-could-rebrand-it-as-macos

Also with this the idea would be keep the platform names consistent with the user facing naming schemes. There are some devs that wouldn't know in the iOS Dev portal that tvOS is really a sub platform.